### PR TITLE
chore(documentation): Add PR Template

### PR DESCRIPTION
**Pull request by westeezy**
First swing at a basic and generic PR template. If this looks good I'll add to all relevant public repos.

### CODE REVIEW

**Review by ssono — APPROVED** [review]
Could the what, why, how be their own sections in place of description? Feels more explicit and will be more prominent?

**Comment by westeezy** [comment]
@ssono thats a great suggestion! I wanted to be careful about being too prescriptive, but that sounds like a good balance to me.

**Review by elizabethmv — APPROVED** [review]
nice! non blocking suggestion: I like to see steps to reproduce in the description for those that want to pull the changes down and confirm the behavior locally.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+### Your checklist for this pull request
+🚨 Please review the [guidelines for contributing](../../CONTRIBUTING.md) to this repository.
```
Comment (gregjopa) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: I think we need to pump the brakes here @westeezy. 🚨 is the police car light emoji. Reviewing the guides as a pre-req is not an emergency. We don't want flashing lights to scare off future contributors.

How about 🔬 or 🍏 instead?

Comment (bluepnume) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: 🦆 please

Comment (westeezy) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: i like a little intimidation in my prs. 🦁

**Comment by westeezy** [comment]
> nice! non blocking suggestion: I like to see steps to reproduce in the description for those that want to pull the changes down and confirm the behavior locally.

@elizabethmv love it. I'll add that in

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (gregjopa) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: The lion is pretty sweet too 🦁. We could use that

**Pull request merged**